server/data/incidentReportingApi.ts

```diff
@@ -473,6 +473,13 @@ export class IncidentReportingApi extends RestClient {
     })
     return questions.map(convertQuestionDates)
   }
+
+  async deleteLastQuestionWithResponses(reportId: string): Promise<Question[]> {
```

Comment:
perhaps a slightly clearer name for this method could be `deleteLastQuestionAndItsResponses()`...more accurate description of what's doing, may help people not-so familiar with the API.

...but maybe pedantic?